core/services/pipeline/runner.go

```diff
@@ -180,17 +175,13 @@ func (r *runner) ResultsForRun(ctx context.Context, runID int64) ([]Result, erro
 // NOTE: This could potentially run on a different machine in the cluster than
 // the one that originally added the job run.
 func (r *runner) processUnfinishedRuns() {
-	_, err := r.processRun()
-	if err != nil {
-		logger.Errorf("Error processing unfinished run: %v", err)
-	}
-}
-
-func (r *runner) processRun() (anyRemaining bool, err error) {
```

Comment:
this function was actually a superfluous leftover from when we executed things at the task level